ouroboros-consensus/src/Ouroboros/Consensus/Network/NodeToNode.hs

```diff
@@ -679,7 +612,7 @@ mkApps kernel Tracers {..} mkCodecs ByteLimits {..} genChainSyncTimeout ReportPe
         blTxSubmission2
         timeLimitsTxSubmission2
         channel
-        (Hello.wrapServerPeerPipelined (txSubmissionServerPeerPipelined (hTxSubmissionServer version them)))
+        (txSubmissionServerPeerPipelined (hTxSubmissionServer version them))
```

Comment:
I assuming you folded the `Hello` wrapper into the underlying definitions, and that's why it's removed here? @coot

Reply:
Yes, that's right.  I first implemented `TxSubmission2` without `Hello` transformer, and I wrote a [test](https://github.com/input-output-hk/ouroboros-network/pull/3696/commits/066587982f05ac2a8775e78b7285f3604cdca70f?show-viewed-files=true&file-filters%5B%5D=#diff-45bb0d334c47f8ebc5f9744484a8e7944805499bc137a53e36d62c578c14b709R458-R463) which shows that the codec is the same, it was removed together with the `Hello` wrapper.